### PR TITLE
About Us Page

### PR DESCRIPTION
**Pull request by jejeromemngls15**
# Added About Us Page

## Details
- Created the draft for all the sections of About Us page.

## Juxtaposition

image
image
image
image
image
image

## Additional Notes

- Page is not yet responsive 
- Map is not yet integrated
- No Footer

### CODE REVIEW

File: src/app/about-us/page.tsx
Comment (carlmitzchel) [inline src/app/about-us/page.tsx:1]: Might consider using `wave component` separately to enhance quality.

image

Comment (carlmitzchel) [reply src/app/about-us/page.tsx:1]: Experiment with tailwind's `leading-` to minimize vertical spaces in sentences. 
image

Comment (carlmitzchel) [reply src/app/about-us/page.tsx:1]: Use the `barrel` method for importing and exporting components. You can check `/components` folder in the main branch.

**Review by carlmitzchel — CHANGES_REQUESTED** [review]

**Review by JoshuaEntrata — COMMENTED** [review]
create index file for components. refer to the components used in Amenities and Offers pages

File: src/app/about-us/page.tsx
Comment (jejeromemngls15) [reply src/app/about-us/page.tsx:1]: okay na yung barrel and tailwind leading. not sure with the wave component if okay yung approach ko, pacheck nalng

**Review by carlmitzchel — APPROVED** [review]

**Pull request merged**